implementations/rust/daemon/src/cli.rs

```diff
@@ -241,19 +257,23 @@ impl FromStr for VaultKind {
 pub enum ChannelRole {
     /// The Initiator role expects a channel responder address and a public key to use in order to
     /// communicate with the Responder end of the channel.
-    Initiator,
+    Source,
     /// The Responder role will create a channel responder, and will instruct the program to print
     /// the responder's channel responder address and the public key it's advertising.
-    Responder,
+    Sink,
+    /// The Router role will route messages along the path, and is capable of acting as a responder
+    /// to establish channels for tunneling
+    Router,
 }
 
 impl FromStr for ChannelRole {
     type Err = String;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
-            "initiator" | "init" => Ok(ChannelRole::Initiator),
-            "responder" | "resp" => Ok(ChannelRole::Responder),
+            "initiator" | "init" => Ok(ChannelRole::Source),
+            "responder" | "resp" => Ok(ChannelRole::Sink),
+            "router" => Ok(ChannelRole::Router),
```

Comment:
these should also reflect the naming change (again, if we're going to commit to it & @mrinalwadhwa approves).. this particular function takes the string literal from the CLI input for the `--role` arg, and translates it into the proper enum variant. So we want these to be consistent as well.

Reply:
Please help me understand the thought behind this?

Reply:
@mrinalwadhwa The "responder" role is no longer really appropriate for the sink end of the pipe (i.e. the end that pumps data into Influx). The sink acts as initiator when it establishes the channel with the router, and as responder when it gets M1 from the source end of the pipe. "Source" and "sink" are less ambiguous IMO and don't muddy the initiator/responder relationship.

Reply:
done